plugins/node/opentelemetry-instrumentation-express/test/hooks.test.ts

```diff
@@ -169,4 +169,140 @@ describe('ExpressInstrumentation hooks', () => {
       );
     });
   });
+
+  describe('span attributes hooks', () => {
+    let server: http.Server;
+    let port: number;
+    let rootSpan: Span;
+
+    beforeEach(async () => {
+      rootSpan = tracer.startSpan('rootSpan');
+
+      const httpServer = await serverWithMiddleware(tracer, rootSpan, app => {
+        app.get('*', (req, res) => {
+          res.send('ok');
+        });
+      });
+      server = httpServer.server;
+      port = httpServer.port;
+    });
+
+    afterEach(() => {
+      server.close();
+    });
+
+    it('should set span attributes', async () => {
+      instrumentation.setConfig({
+        spanAttributesHook: ({ request, route, layerType }, baseAttributes) => {
+          if (layerType) {
+            return {
+              route,
+            };
+          }
+
+          if (route === '*') {
+            return {
+              method: request.method,
+              url: request.url,
+            };
+          }
+
+          return baseAttributes;
```

Comment:
Is this path being invoked in the test?

Reply:
It isn't. I reused it from the span name hook. The tests that could use it set their own hook. So I've removed it.

Reply:
Even though it isn't used/usable, it's actually required to get the linter to pass.